### DISCUSSION TITLE
Wiki Sidebar

### DISCUSSION THREAD

**Discussion by IHateYourCode** [texstudio-org/texstudio#1805] [Q&A] [answered]
[shows the exact same link]: https://github.com/texstudio-org/texstudio/wiki

Was there a specific reason why the wiki sidebar contains
`https://github.com/texstudio-org/texstudio.wiki.git`
or is it an old relic?

That is, considering **Github** [shows the exact same link] (for cloning) by default.
> Clone this wiki locally

**Comment by sunderme** [comment]
I am not aware of that link.

**Reply by IHateYourCode** [reply]
It's in the custom wiki `_Sidebar.md` file

![sidebar](https://user-images.githubusercontent.com/89636625/136081965-353b7792-26b9-49b6-b438-dd2e3965a199.png)

**Comment by sunderme** [comment, marked as answer]
I would assume it a relic.
